packages/strapi-parts/src/Tabs/Tabs.js

```diff
@@ -97,25 +108,28 @@ export const Tab = ({ selected, id, children, variant, hasError, onClick, onTabC
       textColor = 'danger600';
     } else if (selected) {
       textColor = 'primary600';
+    } else if (disabled) {
+      textColor = 'neutral600';
     } else {
       textColor = 'neutral600';
     }
 
     return (
-      <button
+      <TabButton
         id={tabId}
         role="tab"
         aria-controls={selected ? tabPanelId : undefined}
         tabIndex={selected ? 0 : -1}
         aria-selected={selected}
         type="button"
         onClick={handleClick}
+        aria-disabled={disabled}
```

Comment:
Don't we need to pass the disabled props also ?

```js
disabled={disabled}
```

Reply:
disabled=disabled is not practical for people using keyboard. When using disabled, the field is not tabbable and so not announced to the users. On the other hand, relying on aria-disabled and preventing the real click allows for the button to be tabbable and announced as "dimmed" by SRs